README.md

```diff
@@ -46,6 +46,50 @@ The editor includes a formatting toolbar.
 
 This component is Apache-2 licensed Open Source. Contributors welcome!
 
+### Keyboard shortcuts
+
+The keyboard shortcuts are as follows
+
+Windows/Linux:
+
+- Common actions
+  - `CTRL` + `C` - Copy
+  - `CTRL` + `X` - Cut
+  - `CTRL` + `V` - Paste
+  - `CTRL` + `Z` - Undo
+  - `CTRL` + `SHIFT` + `Z` - Redo
+  - `CTRL` + `Y` - Repeat last action
+
+- Text formatting
+  - `CTRL` + `B` - Bold
+  - `CTRL` + `I` - Italicize 
+  - `CTRL` + `SHIFT` + `.` - Block quotes
+  - `CTRL` + `SHIFT` + `9` - Code
+
+- Paragraph formatting
+  - `CTRL` + `SHIFT` + `7` - Numbered list
+  - `CTRL` + `SHIFT` + `8` - Bulleted list
+
+Mac:
+
+- Common actions
+  - `⌘` + `C` - Copy
+  - `⌘` + `X` - Cut
+  - `⌘` + `V` - Paste
+  - `⌘` + `Z` - Undo
+  - `⌘` + `SHIFT` + `Z` - Redo
+  - `⌘` + `Y` - Repeat last action
```

Comment:
I don't think this is a current shortcut we are supporting, we can remove it.

Reply:
this code snippet shows four shortcuts on my screen, are you referring to just "Repeat last action"?

Reply:
Yes sorry I highlighted line 81 but it shows above that line.

Reply:
I've removed it, Please check it now.